src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,128 @@
+    private static final Long THIRD_ID = 3L;
+    private static final int FOURTH_DRIVER = 3;
+    private static final int FIRST_DRIVER = 1;
+    private static final int FIRST_MANUFACTURER = 0;
+    private static final int SECOND_MANUFACTURER = 1;
+    private static final String SEP = System.lineSeparator();
+    private static final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+
     public static void main(String[] args) {
-        // test your code here
+        List<Manufacturer> manufacturers = testManufacturerService();
+        List<Driver> drivers = testDriverService();
+        testCarService(manufacturers, drivers);
+        // UNCOMMENT METHOD BELOW WITH CAUTION
+        //clearAll();
+    }
+
+    private static void testCarService(List<Manufacturer> manufacturers, List<Driver> drivers) {
+        System.out.println(SEP + "CAR TEST SERVICE" + SEP);
+        Car amarok = carService.create(
+                new Car("Amarok",
+                        manufacturers.get(FIRST_MANUFACTURER),
+                        drivers));
+        Car santaFe = carService.create(
+                new Car("Santa Fe",
+                        manufacturers.get(SECOND_MANUFACTURER),
+                        drivers.stream().limit(FIRST_DRIVER).collect(Collectors.toList())));
+        System.out.println(carService.get(SECOND_ID));
+        carService.addDriverToCar(drivers.get(FOURTH_DRIVER), santaFe);
+        carService.getAll().forEach(System.out::println);
+        carService.getAllByDriver(FIRST_ID).forEach(System.out::println);
+        System.out.println(carService.get(SECOND_ID));
+        carService.removeDriverFromCar(drivers.get(FOURTH_DRIVER), santaFe);
+        System.out.println(carService.get(SECOND_ID));
+        carService.delete(SECOND_ID);
+        carService.getAll().forEach(System.out::println);
+    }
+
+    private static List<Driver> testDriverService() {
+        System.out.println(SEP + "DRIVER TEST SERVICE" + SEP);
+        List<Driver> drivers = new ArrayList<>();
+        Driver bob = driverService.create(
+                new Driver("Bob", "NY12345678"));
+        drivers.add(bob);
+        Driver john = driverService.create(
+                new Driver("John", "FL12345678"));
+        drivers.add(john);
+        Driver alice = driverService.create(
+                new Driver("Alice", "NE12345678"));
+        System.out.println(driverService.get(SECOND_ID));
+        System.out.println(driverService.get(THIRD_ID));
+        Driver aliceToEmma = driverService.update(
+                new Driver(THIRD_ID, "Emma", "TE12345678"));
+        drivers.add(aliceToEmma);
```

Comment:
wrong understanding of method update()

after creation id is set to your object, and you can use it in code
and you also have object which you can change (set another name, for example)
than you should call update() method with this updated object (and not pass the new object there!)
also, no need to call add() one more time in the end

```suggestion
        alice.setName("Emma");
        Driver aliceToEmma = driverService.update(alice);
```